style.css

```diff
@@ -259,3 +265,104 @@ h2 {
 .underline {
   list-style: none;
 }
+
+.contactMe {
+  background-color: #6070ff;
+  border-radius: 5rem 0 0 0;
+  background-image: url(./img/background/contactFormBack.svg);
+  background-repeat: no-repeat;
+  background-position: right 0;
+  background-size: 50vw;
+  background-position-x: 12rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  size: 100vw;
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 2.5rem;
+  margin-top: 10rem;
+}
+
+#contactForm {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 90vw;
+  margin-right: 2rem;
+}
+
+#contactP {
+  color: white;
+  width: 85vw;
+  text-align: center;
+  line-height: 28px;
+  font-size: 1.25rem;
+  margin-bottom: 1rem;
+}
+
+#contactTitle {
+  color: #ebebff;
+  text-align: center;
+  font-size: 2.5rem;
+  margin-top: 5rem;
+}
+
+#yourName {
+  font-size: 1.1rem;
+  color: #fe270a;
+  line-height: 20px;
+  border-radius: 5px;
+  border: 1px solid #cfd8dc;
+  height: 3rem;
+  width: 90vw;
+}
```

Comment:
- [ ] Kindly adhere to the project requirement because as I can see you are not using the correct font color. which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached image and table.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-07-16 00-29-28](https://github.com/DamianDib9/personal-portfolio/assets/66476329/0cef64e8-1f55-487c-9b52-0d568a48c1cd)

</td>
<td>

![Screenshot from 2023-07-16 00-26-55](https://github.com/DamianDib9/personal-portfolio/assets/66476329/0b401170-af75-4073-b2b6-dedc4e31932b)

</td>
</tr>
</table>

Reply:
![image](https://github.com/DamianDib9/personal-portfolio/assets/121461459/8cec40b4-2345-4f1b-85a3-dbe1b0a26c32)

OK!
Done